eternalcore-core/src/main/java/com/eternalcode/core/translation/Translation.java

```diff
@@ -362,6 +362,13 @@ interface LanguageSection {
         Notice languageChanged();
     }
 
+    // Placeholder section
+    PlaceholderSection placeholder();
+
+    interface PlaceholderSection {
+        String homeNotFound();
+    }
```

Comment:
It seems to me that it should rather be in the home section. When someone uses, for example, only the home system, they configure only this section. In this case, he will have to scroll through one large configuration file